app/assets/javascripts/components/common/wikidata_overview_stats.jsx

```diff
@@ -3,10 +3,16 @@ import PropTypes from 'prop-types';
 import OverviewStat from './OverviewStats/overview_stat';
 import I18n from 'i18n-js';
 
-const WikidataOverviewStats = ({ statistics }) => {
+const WikidataOverviewStats = ({ statistics, isCourseOverview }) => {
+  let containerClass = 'wikidata-stats-container';
+  let title = 'Wikidata stats';
```

Comment:
It looks like this title isn't used in the the latest screenshots you shared?

Reply:
Yes, it is used. In the last one. Here, the title isn't "Wikidata stats", but "www.wikidata.org". I thought that this would be in accordance with other namespace stats and their title. So, "Wikidata Stats" would be only shown for campaign overview stats. Is it okay?